application/features/email.feature

```diff
@@ -205,11 +204,38 @@ Feature: Email
       | to send            | does exist     | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does exist      | delete_mfa   | should NOT  |
       | to send            | does exist     | does exist      | delete_mfa   | should NOT  |
-      | to send            | does NOT exist | does NOT exist  | delete_mfa   | should      |
       | NOT to send        | does NOT exist | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
       | NOT to send        | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
 
+
+  Scenario Outline: When to send an email after a verified u2f mfa option has been deleted.
+    Given we are configured to send mfa disabled emails
+      And we are configured to send mfa option removed emails
+      And no mfas exist
+      And a user already exists
+      And a verified u2f mfa option used to exist
+      And the latest mfa event type was delete_mfa
+    When I check if a mfa <optionRemovedOrDisabled> email should be sent
+    Then I see that a mfa <optionRemovedOrDisabled> email <shouldOrNot> be sent
+
+    Examples:
+      | optionRemovedOrDisabled | shouldOrNot |
+      | option removed          | should NOT  |
+      | disabled                | should      |
+
+  Scenario: When to send an email after an unverified u2f mfa option has been deleted.
+    Given we are configured to send mfa disabled emails
+      And we are configured to send mfa option removed emails
+      And no mfas exist
+      And a user already exists
+      And an unverified u2f mfa option does exist
+      And a verified u2f mfa option does exist but not as the test mfa
+      And the latest mfa event type was delete_mfa
+    When I check if a mfa option removed email should be sent
```

Comment:
Based on the wording of this test scenario, I don't think it's describing the situation we're trying to test here. (For example, it doesn't seem to say whether the MFA option just deleted was verified or not... simply that there is an unverified MFA option and a verified MFA option that do exist).

What if we changed from these lines...
```gherkin
    And an unverified u2f mfa option does exist
    And a verified u2f mfa option does exist but not as the test mfa
    And the latest mfa event type was delete_mfa
```
... to this:
```gherkin
    And a verified u2f mfa option does exist
    And an unverified u2f mfa option was just deleted
```